recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
I would try to get rid of those rather by addition of `-DBUILD_GUI=OFF`, than including them. Can you point me to the travis job building OB packages?